### ISSUE TITLE
The list of pq shows Active and Ready for Download together of one page

### ISSUE THREAD

**Issue by 2Abendsegler** [2Abendsegler/GClh#830]
**Describe the bug**
The list of pq shows Active and Ready for Download together of one page in compact layout.

**To Reproduce**
1. Go to https://www.geocaching.com/pocket/default.aspx

**Screenshots**
![x1](https://user-images.githubusercontent.com/22332216/45003761-f7872180-afe5-11e8-9ccd-e849d73ed43a.JPG)

**Desktop (please complete the following information):**
 - Windows 8
 - FF 61.0.1 (64-Bit)
 - TM 4.7.5788

**Comment by Ruko2010** [comment]
@2Abendsegler why is this a bug? I would consider it a feature! I saw it yesterday testing the PQ-generator and I liked it. So you "save" a click if you want to download a PQ. Maybe you revert your change and make it configurable? Add a new Config-Value: "Show Downloadable PQs on first page in compact layout" or something like that.

**Comment by Ruko2010** [comment]
Of cause you have to hide the link to "PQ ready for download" if the option is enabled.

**Comment by 2Abendsegler** [comment]
Ja, hatte ich auch zuerst gedacht, so schlecht ist nicht. Aber das untere Teil ist nicht stabil, wenn man oben die Tabs anwählt, ist es dann auch weg.

Vielleicht können wir es zuerst mal so lassen und das Feature für die nächste Version vorsehen?

Bin jetzt nich so schnell. Muss jetzt wandern gehen.

**Comment by Ruko2010** [comment]
Na klar, kein Problem. Du weist doch 
Wandern > GClh 😉

**Comment by 2Abendsegler** [comment]
Ich bau die List of PQ noch um, falls du noch so viel Zeit hast. Bin schon fast fertig.

**Comment by Ruko2010** [comment]
klar, ich erledige erstmal die Changelogs.
